openhtf/core/station_api.py

```diff
@@ -453,7 +453,7 @@ def history(self):
     """
     last_start_time = self._cached_history.last_start_time(self.test_uid)
     new_history = self.shared_proxy.get_history_after(
-        self.test_uid, long(last_start_time))
+        self.test_uid, int(last_start_time))
```

Comment:
I think 2to3 did this, but to preserve the intent of the original code explicitly requesting wider-than-integer size, I think we need `from builtins import int` at the top of this file.

Alternately we can revert these long => int changes and add `from past.builtins import long`

http://python-future.org/compatible_idioms.html#long-integers